roles/flannel_register/defaults/main.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-flannel_network: "{{ openshift.master.portal_net | default('172.30.0.0/16', true) }}"
+flannel_network: "{{ openshift.common.portal_net | default('172.30.0.0/16', true) }}"
```

Comment:
If we are going to switch this from master/node to common, we'll need to write a migration script to do it.

I started to implement this here: https://github.com/detiber/openshift-ansible/blob/openshift_facts/roles/openshift_facts/library/openshift_facts.py#L1215-L1241 but never finished cleaning up that PR to go further with it.